Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -98,7 +94,7 @@ public class OrnamentsManager: NSObject {
         // Scalebar View
         // Check whether the scale bar is position on the right side of the map view.
         let scaleBarPosition = options.scaleBar.position
-        scaleBarView.isOnRight = scaleBarPosition == .bottomRight || scaleBarPosition == .topRight
+        scaleBarView.isOnRight = scaleBarPosition == .bottomRight || scaleBarPosition == .bottomTrailing || scaleBarPosition == .topRight
```

Comment:
Should this also include `.topTrailing`?

Reply:
Nope, reverted changes here because we just need to remove the deprecation (rather than impacting the functionality). Same for the other comment.